tests/docker.sipnet.xml

```diff
@@ -0,0 +1,67 @@
+<?xml version="1.0"?>
+<pecan>
+  <outdir>/data/tests/sipnet</outdir>
+
+  <database>
+    <bety>
+      <driver>PostgreSQL</driver>
+      <user>bety</user>
+      <password>bety</password>
+      <host>postgres</host>
+      <dbname>bety</dbname>
+      <write>FALSE</write>
+    </bety>
+  </database>
+
+  <pfts>
+    <pft>
+      <name>temperate.coniferous</name> 
+    </pft>
+  </pfts>
+
+  <meta.analysis>
+    <iter>3000</iter>
+    <random.effects>FALSE</random.effects>
+    <threshold>1.2</threshold>
+    <update>AUTO</update>
+  </meta.analysis>
+
+  <ensemble>
+    <variable>NPP</variable>
+  </ensemble>
+
+  <sensitivity.analysis>
+    <quantiles>
+      <sigma>-1</sigma>
+      <sigma>1</sigma>
+    </quantiles>
+    <variable>NPP</variable>
+  </sensitivity.analysis>
+
+  <model>
+    <type>SIPNET</type>
+    <revision>r136</revision>
+  </model>
+
+  <run>
+    <site>
+      <id>772</id>
+    </site>
+    <inputs>
+      <met>
+        <id>5000000005</id>
```

Comment:
for me the id is `99000000003` and I have no record in `localhost:8000/bety/inputs/5000000005`

Reply:
I think you are missing the data from Ankur, if you have no data yet, can you try to initialize the database once more from the pecan/db:latest. (make sure you do a `docker pull pecan/db` before).

Reply:
```
docker pull pecan/db

Using default tag: latest
latest: Pulling from pecan/db
Digest: sha256:5cb1c84ac2afaa3e7e4f78fbd6e55d5d24822cea6b0e29294df3014173c41212
Status: Image is up to date for pecan/db:latest
docker.io/pecan/db:latest

 docker run -ti --rm --network pecan_pecan --volume pecan_pecan:/data --env FQDN=docker pecan/data:develop

######################################################################
CREATE FOLDERS
######################################################################
######################################################################
Adding sites
######################################################################
######################################################################
Adding inputs
######################################################################
######################################################################
Adding LPJ-Guess CRU
######################################################################
######################################################################
Adding Plot
######################################################################
######################################################################
Adding Santarem_Km83
######################################################################
######################################################################
Adding testrun.s83
######################################################################
######################################################################
Adding ed2ws.harvard
######################################################################
######################################################################
Adding testrun.PDG
######################################################################
######################################################################
Adding create_met_driver
######################################################################
######################################################################
REGISTER DATA
######################################################################
Found input with ID= 294 for site=1118, format_id=41, start=1800-01-01 05:50:36, end=1999-01-01 06:00:00
File /data/ed_inputs/glu/ already added to input  294 on host docker.
Found input with ID= 295 for site=1118, format_id=42, start=, end=
File /data/ed_inputs/ already added to input  295 on host docker.
Found input with ID= 296 for site=1118, format_id=43, start=, end=
File /data/oge2OLD/OGE2_ already added to input  296 on host docker.
Found input with ID= 297 for site=1118, format_id=44, start=, end=
File /data/faoOLD/FAO_ already added to input  297 on host docker.
Found input with ID= 1000000903 for site=1118, format_id=1000000020, start=, end=
File /data/lpj-guess/cru_1901_2006.bin already added to input  1000000903 on host docker.
Found input with ID=  5 for site=76, format_id=10, start=, end=
File /data/sites/ebifarm/ebifarm.lat40.0lon-88.0.site already added to input   5 on host docker.
Found input with ID=  6 for site=76, format_id=11, start=, end=
File /data/sites/ebifarm/ebifarm.lat40.0lon-88.0.css already added to input   6 on host docker.
Found input with ID=  7 for site=76, format_id=12, start=2004-01-01 00:00:00, end=2009-12-31 23:59:59
File /data/sites/ebifarm/ED_MET_DRIVER_HEADER already added to input   7 on host docker.
Found input with ID=  8 for site=76, format_id=15, start=, end=
File /data/sites/ebifarm/ebifarm.lat40.0lon-88.0.pss already added to input   8 on host docker.
Found input with ID= 82 for site=758, format_id=12, start=1991-01-01 06:00:00, end=2006-12-31 06:00:00
File /data/sites/harvard_ems/ED_MET_DRIVER_HEADER already added to input  82 on host docker.
Found input with ID= 83 for site=758, format_id=16, start=1991-01-01 06:00:00, end=2006-12-31 06:00:00
File /data/sites/harvard_ems/US-Ha1forcing.nc already added to input  83 on host docker.
Found input with ID= 136 for site=758, format_id=10, start=, end=
File /data/sites/harvard_ems/harvard.NACP.lat42.5lon-72.5.site already added to input  136 on host docker.
Found input with ID= 138 for site=758, format_id=11, start=, end=
File /data/sites/harvard_ems/harvard.NACP.lat42.5lon-72.5.css already added to input  138 on host docker.
Found input with ID= 137 for site=758, format_id=15, start=, end=
File /data/sites/harvard_ems/harvard.NACP.lat42.5lon-72.5.pss already added to input  137 on host docker.
Found input with ID= 2000000001 for site=676, format_id=24, start=1998-01-01, end=2006-12-31
File /data/sites/willow/wcr.clim already added to input  2000000001 on host docker.
Found input with ID= 134 for site=676, format_id=12, start=1998-01-01 06:00:00, end=2006-12-31 06:00:00
File /data/sites/willow/ED_MET_DRIVER_HEADER already added to input  134 on host docker.
Found input with ID= 135 for site=676, format_id=16, start=1998-01-01 06:00:00, end=2006-12-31 06:00:00
File /data/sites/willow/US-WCrforcing.nc already added to input  135 on host docker.
Found input with ID= 171 for site=676, format_id=10, start=1998-01-01 06:00:00, end=2006-12-31 06:00:00
File /data/sites/willow/WCr.NACP.lat45.5lon-90.5.site already added to input  171 on host docker.
Found input with ID= 169 for site=676, format_id=11, start=1998-01-01 06:00:00, end=2006-12-31 06:00:00
File /data/sites/willow/WCr.NACP.lat45.5lon-90.5.css already added to input  169 on host docker.
Found input with ID= 170 for site=676, format_id=15, start=1998-01-01 06:00:00, end=2006-12-31 06:00:00
File /data/sites/willow/WCr.NACP.lat45.5lon-90.5.pss already added to input  170 on host docker.
Found input with ID= 171 for site=676, format_id=10, start=1998-01-01 06:00:00, end=2006-12-31 06:00:00
File /data/sites/willow/US-WCr.Inv.lat45.5lon-90.site already added to input  171 on host docker.
Found input with ID= 169 for site=676, format_id=11, start=1998-01-01 06:00:00, end=2006-12-31 06:00:00
File /data/sites/willow/US-WCr.Inv.lat45.5lon-90.css already added to input  169 on host docker.
Found input with ID= 170 for site=676, format_id=15, start=1998-01-01 06:00:00, end=2006-12-31 06:00:00
File /data/sites/willow/US-WCr.Inv.lat45.5lon-90.pss already added to input  170 on host docker.
Found input with ID= 2000000002 for site=622, format_id=24, start=2001-01-01, end=2006-12-31
File /data/sites/sylvana/syl.clim already added to input  2000000002 on host docker.
Found input with ID= 132 for site=622, format_id=12, start=2001-01-01 06:00:00, end=2006-12-31 06:00:00
File /data/sites/sylvana/ED_MET_DRIVER_HEADER already added to input  132 on host docker.
Found input with ID= 133 for site=622, format_id=16, start=2001-01-01 06:00:00, end=2006-12-31 06:00:00
File /data/sites/sylvana/US-Syvforcing.nc already added to input  133 on host docker.
Found input with ID= 177 for site=622, format_id=10, start=2001-01-01 06:00:00, end=2006-12-31 06:00:00
File /data/sites/sylvana/Syl.NACP.lat46.5lon-89.5.site already added to input  177 on host docker.
Found input with ID= 175 for site=622, format_id=11, start=2001-01-01 06:00:00, end=2006-12-31 06:00:00
File /data/sites/sylvana/Syl.NACP.lat46.5lon-89.5.css already added to input  175 on host docker.
Found input with ID= 176 for site=622, format_id=15, start=2001-01-01 06:00:00, end=2006-12-31 06:00:00
File /data/sites/sylvana/Syl.NACP.lat46.5lon-89.5.pss already added to input  176 on host docker.
Found input with ID= 99000000001 for site=678, format_id=24, start=1997-01-01, end=2005-12-31
File /data/sites/parkfalls/lef.clim already added to input  99000000001 on host docker.
Found input with ID= 130 for site=678, format_id=12, start=1995-01-01 06:00:00, end=2005-12-31 06:00:00
File /data/sites/parkfalls/ED_MET_DRIVER_HEADER already added to input  130 on host docker.
Found input with ID= 131 for site=678, format_id=16, start=1995-01-01 06:00:00, end=2005-12-31 06:00:00
File /data/sites/parkfalls/US-PFaforcing.nc already added to input  131 on host docker.
Found input with ID= 99000000002 for site=679, format_id=24, start=2001-01-01, end=2006-12-31
File /data/sites/lostcreek/lcr.clim already added to input  99000000002 on host docker.
Found input with ID= 92 for site=679, format_id=12, start=2000-01-01 06:00:00, end=2006-12-31 06:00:00
File /data/sites/lostcreek/ED_MET_DRIVER_HEADER already added to input  92 on host docker.
Found input with ID= 93 for site=679, format_id=16, start=2000-01-01 06:00:00, end=2006-12-31 06:00:00
File /data/sites/lostcreek/US-Losforcing.nc already added to input  93 on host docker.
Found input with ID= 99000000003 for site=772, format_id=24, start=2002-01-01, end=2005-12-31
File /data/sites/niwot/niwot.clim already added to input  99000000003 on host docker.
Found input with ID= 99000000004 for site=772, format_id=24, start=1999-01-01, end=2003-12-31
File /data/sites/niwot/US-NR1.clim already added to input  99000000004 on host docker.
Found input with ID= 112 for site=772, format_id=12, start=1998-01-01 06:00:00, end=2008-01-01 05:00:00
File /data/sites/niwot/ED_MET_DRIVER_HEADER already added to input  112 on host docker.
Found input with ID= 113 for site=772, format_id=16, start=1998-01-01 06:00:00, end=2007-12-31 06:00:00
File /data/sites/niwot/US-NR1forcing.nc already added to input  113 on host docker.
Found input with ID= 188 for site=772, format_id=10, start=1998-01-01 06:00:00, end=2007-12-31 06:00:00
File /data/sites/niwot/NR1.NACP.lat40.5lon-105.5.site already added to input  188 on host docker.
Found input with ID= 186 for site=772, format_id=11, start=1998-01-01 06:00:00, end=2007-12-31 06:00:00
File /data/sites/niwot/NR1.NACP.lat40.5lon-105.5.css already added to input  186 on host docker.
Found input with ID= 187 for site=772, format_id=15, start=1998-01-01 06:00:00, end=2007-12-31 06:00:00
File /data/sites/niwot/NR1.NACP.lat40.5lon-105.5.pss already added to input  187 on host docker.
Found input with ID= 2000000004 for site=766, format_id= 1000000002, start=1999-01-01, end=2003-12-31
File /data/dalec_EnKF_pub/input_data/oregon/dalec_drivers.OREGON.no_obs.dat already added to input  2000000004 on host docker.
######################################################################
Done!
######################################################################

```

Reply:
I just did `docker-compose run bety sync` and got synced with 0 1 2. Couldn't synced with 5 (which I assume is Wisconsin) 

```
docker-compose run bety sync

Starting pecan_postgres_1 ... done
Synchronize with servers 0 1 2 5
Synchronizing with server 0
URL with bety dump                 : https://ebi-forecast.igb.illinois.edu/pecan/dump/bety.tar.gz
Remote start ID                    :           0
Remote end ID                      :   999999999
Local start ID                     : 99000000001
Local end ID                       : 99999999999
Synchronizing with server 1
URL with bety dump                 : https://psql-pecan.bu.edu/sync/dump/bety.tar.gz
Remote start ID                    :  1000000001
Remote end ID                      :  1999999999
Local start ID                     : 99000000001
Local end ID                       : 99999999999
Synchronizing with server 2
URL with bety dump                 : ftp://anon:anon@ftp.test.bnl.gov/outgoing/betydb/bety.tar.gz
Remote start ID                    :  2000000001
Remote end ID                      :  2999999999
Local start ID                     : 99000000001
Local end ID                       : 99999999999
Synchronizing with server 5
Did not find a sync_url in database, please update database, or provide script with DUMPURL.

```
I can see records from bnl, e.g. `http://localhost:8000/bety/formats/2000000002` exists but not `http://localhost:8000/bety/formats/5000000002`

Reply:
Sorry, not sure what's going on with  `5000000005` but assuming the `docker run -ti --rm --network pecan_pecan --volume pecan_pecan:/data --env FQDN=docker pecan/data:develop` command above is fixed, changing this id to be `99000000003` on the docker.sipnet.xml does it, maybe for now we can resort to that so that this PR can go in